sbin/build.sh

```diff
@@ -241,9 +246,11 @@ configureVersionStringParameter() {
   if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" != 8 ]; then
     addConfigureArg "--with-vendor-name=" "\"${BUILD_CONFIG[VENDOR]}\""
   fi
-  addConfigureArg "--with-vendor-url=" "${BUILD_CONFIG[VENDOR_URL]:-"https://adoptium.net/"}"
-  addConfigureArg "--with-vendor-bug-url=" "${BUILD_CONFIG[VENDOR_BUG_URL]:-"https://github.com/adoptium/adoptium-support/issues"}"
-  addConfigureArg "--with-vendor-vm-bug-url=" "${BUILD_CONFIG[VENDOR_VM_BUG_URL]:-"https://github.com/adoptium/adoptium-support/issues"}"
+  addConfigureArg "--with-vendor-url=" "${BUILD_CONFIG[VENDOR_URL]:-""}"
+  # This looks silly, but in the case where someone is building a plain hotspot
+  # This makes it a bit easier to find the code that sets it to override
+  addConfigureArg "--with-vendor-bug-url=" "${BUILD_CONFIG[VENDOR_BUG_URL]:-"file:///dev/null"}"
```

Comment:
Perhaps could even add a sample "Please replace with your own Vendor URL e.g. ....."?

Reply:
My only concern what that is that ideally we wouldn't want people to modify the scripts but use a parameter/variable. but I'll add a comment in for now.